tests/test_results.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pytest
+
+from pydvl.utils.types import SortOrder
+from pydvl.value import ValuationResult, ValuationStatus
+
+
+def dummy_valuator():
+    pass
+
```

Comment:
I think adding a test that makes sure that the objects can be serialized and deserialized without errors would be good idea.

Reply:
A very good one

Reply:
Implemented in https://github.com/appliedAI-Initiative/pyDVL/pull/214/commits/05ac4d1cc48fe6006c975e110ed7966bbb45b03d for pickle and cloudpickle.